### ISSUE TITLE
Improvements to custom select

### ISSUE THREAD

**Issue by Wolfr** [Wolfr/sveltekit-jui#94]
When the select is open, on the parent element, aria-expanded should be set to `true`.

The `c-select-menu__item` list item should have a `aria-selected="true"` whenever it is selected. All items should have a `tabindex` of `0` and a `role="menuitem"`.
